beacon-chain/blockchain/receive_attestation.go

```diff
@@ -93,3 +99,21 @@ func (s *Service) processAttestation() {
 		}
 	}
 }
+
+func (s *Service) verifyCheckpointEpoch(c *ethpb.Checkpoint) bool {
```

Comment:
Can we have comments on this func? It's not clear . at all what this does without having prior context. Maybe also consider a little bit better naming.